### DISCUSSION TITLE
About moodle test server

### DISCUSSION THREAD

**Discussion by Aniketk9211** [Department-of-Chemistry-PU/moodle-tiny_moldraw#2] [General]
Good Morning, Sir
The Moodle Server is not responding. Its show "The site can't be reached." 
One more thing each time i update code its not reflect on server easily. Is their way to solve this.

Thanks.

**Comment by registerme1963** [comment]
https://node108289-env-2576158.cloudjiffy.net/moodle/ is working well

You need to re-build the code after updating/editing using "npx grunt amd --force"
Or open two ssh session windows and in one of the window you can use "grunt watch"

Please update the code in the git repository first and clone it in the server. By this you can keep track of the changes
rV

**Reply by Aniketk9211** [reply]
thanks sir
